tce/src/applibs/PIG/HexImageWriter.cc

```diff
@@ -0,0 +1,90 @@
+/*
+    Copyright (c) 2002-2018 Tampere University of Technology.
```

Comment:
Also add your name to the copyright as you own the code (no copyright assignments to TUT to my knowledge). Tampere University of Technology and [YOURNAME] is probably fair as it uses TUT written interfaces.